packages/docusaurus/lib/commands/eject.js

```diff
@@ -8,16 +8,29 @@
 const fs = require('fs-extra');
 const chalk = require('chalk');
 const path = require('path');
+const importFresh = require('import-fresh');
 
-module.exports = async function eject(siteDir) {
-  const defaultTheme = path.resolve(__dirname, '..', 'default-theme');
-  const customTheme = path.resolve(siteDir, 'theme');
-  await fs.copy(defaultTheme, customTheme);
+module.exports = async function eject(siteDir, pluginName) {
+  let Plugin;
+  try {
+    Plugin = importFresh(pluginName);
+  } catch (ex) {
+    throw new Error(`Error loading '${pluginName}' plugin.`);
```

Comment:
This line doesn't serve much purpose. We should not swallow exceptions if we're not going to handle the error anyway. By doing this, we're only hiding the real reason it's failing to import, and making it harder for developers to debug. While developing plugins and when I encountered problems in this code path, these errors are not helpful at all in helping me debug the problem. I ended up commenting out this line and letting the original exception be thrown to see the real error message, which instantly showed me the problem. I also changed the way we did this in the plugin loading if you didn't realize 😅